### PR TITLE
chore: update importlib-metadata pin

### PR DESCRIPTION
**Pull request by Coniferish**

### CODE REVIEW

File: requirements/ingest/chroma.in [hunk 1]
```diff
@@ -1,6 +1,10 @@
 -c ../deps/constraints.txt
 -c ../base.txt
 chromadb
-importlib-metadata>=7.1.0
+importlib-metadata>=8.2.0
```
Comment (Coniferish) [inline requirements/ingest/chroma.in:4]: `opentelemetry-api` has `importlib-metadata` pinned [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/pyproject.toml) to `"importlib-metadata >= 6.0, <= 8.2.0",`

File: requirements/deps/constraints.txt [hunk 2]
```diff
@@ -57,11 +57,3 @@ fsspec==2024.5.0
 wrapt>=1.14.0
 
 langchain-community>=0.2.5
-
-# NOTE(robinson): choma was pinned to importlib-metadata>=7.1.0 but 7.1.0 was installed
-# instead of 7.2.0. Need to investigate
-importlib-metadata==7.1.0
```
Comment (Coniferish) [inline requirements/deps/constraints.txt:63]: Remove this pin

File: requirements/ingest/chroma.in (hunk 1 above)
Comment (Coniferish) [reply requirements/ingest/chroma.in:4]: Other extras were resolving to 8.2.0, but chroma was resolving to 8.0.0 for some reason. Pinned 8.2.0 as a minimum version to force the chroma dependency to match the others.

**Review by MthwRobinson — APPROVED** [review]
Good to go as long as tests pass.

**Review by scanny — APPROVED** [review]
LGTM

**Pull request merged**